app/services/open_library_service.rb

```diff
@@ -0,0 +1,20 @@
+class OpenLibraryService
+  include HTTParty
+
+  def book_info(isbn)
+    response = HTTParty.get(
+      "https://openlibrary.org/api/books?bibkeys=ISBN:#{isbn}&format=json&jscmd=data"
+    )
+    parser_json_isbn(response.parsed_response, isbn)
+  end
+
+  private def parser_json_isbn(response, isbn)
+    {
+      ISBN: isbn,
+      title: response[response.keys.first]['title'],
```

Comment:
You can save the book info in a variable so as to avoid response[response.keys.first] on each line